.github/workflows/nightly.yaml

```
@@ -11,10 +11,11 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-20.04 ]
+        nodejs: [ 14, 16, 18.12.0 ]
```

This [seems to be supported](https://github.com/actions/setup-node#supported-version-syntax)
```suggestion
        nodejs: [ 14, 16, lts/*, latest ]
```

---

Yes that does make sense given `engines` has no upper bound. Good suggestion! I'll include it when squashing